src/Uno.UWP/Helpers/DeviceHelper.macOS.cs

```diff
@@ -5,6 +5,9 @@ namespace Uno.Helpers
 {
 	internal static class DeviceHelper
 	{
+#if NET_6_0
```

Comment:
- I think the correct name is `NET6_0`?
- Should it be `NET6_0_OR_GREATER` to include .NET 7+ as well?

Reply:
bad habits :) good catch